src/sdk/modules/executors/OwnableExecutor.ts

```diff
@@ -1,198 +1,198 @@
-import {
-  type Account,
-  type Address,
-  type Chain,
-  type Hash,
-  type Hex,
-  type PublicClient,
-  type Transport,
-  type WalletClient,
-  encodeAbiParameters,
-  encodeFunctionData,
-  encodePacked,
-  getAddress,
-  parseAbi
-} from "viem"
-import { SENTINEL_ADDRESS } from "../../account/utils/Constants"
-import { type Holder, toHolder } from "../../account/utils/toHolder"
-import type { NexusClient } from "../../clients/createNexusClient"
-import type { Module } from "../../clients/decorators/erc7579"
-import { BaseExecutionModule } from "../base/BaseExecutionModule"
-import type { Execution } from "../utils/Types"
-export class OwnableExecutorModule extends BaseExecutionModule {
-  public nexusClient: NexusClient
-  public owners: Address[]
-  public override address: Address
+// import {
```

Comment:
Can we just delete files that have comments in? You can leave the code in a different branch if you think it'd be useful later